debian/jessie/foreman/control

```diff
@@ -3,7 +3,7 @@ Maintainer: Greg Sutcliffe <greg.sutcliffe@gmail.com>
 Section: web
 Priority: extra
 Standards-Version: 3.9.1
-Build-Depends: debhelper (>= 7), cdbs, bundler (>= 1.1), ruby2.1-dev, git | git-core,
+Build-Depends: debhelper (>= 7), cdbs, bundler (>= 1.3), ruby2.1-dev, git | git-core,
```

Comment:
There's already bundler 1.7 in Jessie, is there a reason not to use it? Apart from consistency across our packages?